### ISSUE TITLE
onMouseDown doesn't have the event parameter passed in correctly

### ISSUE THREAD

**Issue by anandsainath** [scniro/react-codemirror2#88]
This is the current onMouseDown event that is bound.

```
case 'onMouseDown':
{
  _this.editor.on('mousedown', function(cm) {
    _this.props.onMouseDown(_this.editor, event);
  });
  break;
}
```

The callback function doesn't have an event variable that is passed in - which causes **ReferenceError: event is not defined** in some browsers like Firefox.

**Comment by scniro** [comment]
@anandsainath nice find. Yea now that I'm looking, there's a few which don't explicitly pass the event as well. Will try my best to get a fix in for this soon.

**Comment by anandsainath** [comment]
@scniro I just created a pull request for the same. Can you take a look and check if its good?

**Comment by scniro** [comment]
@anandsainath Fixed with [5.0.3](https://github.com/scniro/react-codemirror2/releases/tag/5.0.3). Thanks for the PR!
